Dockerfile-express

```diff
@@ -0,0 +1,9 @@
+FROM node
+COPY mock_data/ /VisualizationG5/mock_data/
+COPY express/ /VisualizationG5/express/
+WORKDIR /VisualizationG5/express/
+RUN rm -rf node_modules/
```

Comment:
Well, in automatic builds the `node_modules` will not even be present. It's different if you `docker build` the image from source, of course.

 The idea is to have this autobuild, and not managed!